### PR TITLE
CS locale

### PR DESCRIPTION
**Pull request by JozefBiros**
Hello, I tried to create locale for 'cs' language. It's far from perfect but it should work.

### CODE REVIEW

**Comment by JozefBiros** [comment]
Can sombedy review it please?

File: src/locale/cs/index.js [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
+import match from './_lib/match/index.js'
 
 /**
  * @type {Locale}
  * @category Locales
  * @summary Czech locale.
  * @language Czech
  * @iso-639-2 ces
- * @author David Rus [@davidrus]{@link https://github.com/davidrus}
+ * @author David Rus [@davidrusg]{@link https://github.com/davidrus}
```
Comment (kossnocorp) [inline src/locale/cs/index.js]: I'm sure that's a mistake change

File: src/locale/cs/index.js [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
  * @author Pavel Hrách [@SilenY]{@link https://github.com/SilenY}
```
Comment (kossnocorp) [inline src/locale/cs/index.js]: Please also add yourself.

File: src/locale/cs/test.js [hunk 3]
```diff
@@ -1,9 +1,484 @@
...
+    //   })
+    // })
+
+    // it('ordinal time', function () {
+    //   var dateString = '1st hour, 2nd minute, 3rd second'
+    //   var formatString = "ho 'hour', mo 'minute', so 'second'"
+    //   var result = parse(dateString, formatString, baseDate, {locale: locale})
+    //   assert.deepEqual(result, new Date(1986, 3 /* Apr */, 4, 1, 2, 3))
+    // })
+  })
```
Comment (kossnocorp) [inline src/locale/cs/test.js]: Please remove commented code

File: src/locale/cs/_lib/match/index.js [hunk 4]
```diff
@@ -1,47 +1,137 @@
-import buildMatchFn from '../../../_lib/buildMatchFn/index.js'
-import buildParseFn from '../../../_lib/buildParseFn/index.js'
 import buildMatchPatternFn from '../../../_lib/buildMatchPatternFn/index.js'
-import parseDecimal from '../../../_lib/parseDecimal/index.js'
+import buildMatchFn from '../../../_lib/buildMatchFn/index.js'
```
Comment (kossnocorp) [inline src/locale/cs/_lib/match/index.js]: Please localize match code as well before I can merge the PR 👇

File: src/locale/cs/_lib/localize/index.js [hunk 5]
```diff
@@ -1,38 +1,181 @@
...
+}
+
+// v1
+// var timeOfDayValues = {
+//   uppercase: ['DOP.', 'ODP.'],
+//   lowercase: ['dop.', 'odp.'],
+//   long: ['dopoledne', 'odpoledne']
+// }
+
+// v2
```
Comment (kossnocorp) [inline src/locale/cs/_lib/localize/index.js]: Please remove commented code.

File: src/locale/cs/_lib/localize/index.js [hunk 6]
```diff
@@ -1,38 +1,181 @@
...
+  ]
+}
+// v1
+// var weekdayValues = {
+//   narrow: ['ne', 'po', 'út', 'st', 'čt', 'pá', 'so'],
+//   short: ['ned', 'pon', 'úte', 'stř', 'čtv', 'pát', 'sob'],
+//   long: ['neděle', 'pondělí', 'úterý', 'středa', 'čtvrtek', 'pátek', 'sobota']
+// }
+
+// v2
```
Comment (kossnocorp) [inline src/locale/cs/_lib/localize/index.js]: Please remove commented code.

File: src/locale/cs/_lib/localize/index.js [hunk 7]
```diff
@@ -1,38 +1,181 @@
...
+  wide: ['1. čtvrtletí', '2. čtvrtletí', '3. čtvrtletí', '4. čtvrtletí']
 }
 
+// v1
+// var monthValues = {
+//   short: ['led', 'úno', 'bře', 'dub', 'kvě', 'čvn', 'čvc', 'srp', 'zář', 'říj', 'lis', 'pro'],
+//   long: ['leden', 'únor', 'březen', 'duben', 'květen', 'červen', 'červenec', 'srpen', 'září', 'říjen', 'listopad', 'prosinec']
+// }
+
+// v2
```
Comment (kossnocorp) [inline src/locale/cs/_lib/localize/index.js]: Please remove commented code.

File: src/locale/cs/_lib/formatLong/index.js [hunk 8]
```diff
@@ -1,12 +1,53 @@
...
+//   LT: 'h:mm aa',
+//   LTS: 'h:mm:ss aa',
+
+//   L: 'MM/DD/YYYY',
+//   LL: 'MMMM D YYYY',
+
+//   LLL: 'MMMM D YYYY h:mm aa',
+//   LLLL: 'dddd, MMMM D YYYY h:mm aa'
+// })
+
```
Comment (kossnocorp) [inline src/locale/cs/_lib/formatLong/index.js]: Please remove commented code.

File: src/locale/cs/_lib/formatDistance/index.js [hunk 9]
```diff
@@ -1,194 +1,278 @@
...
-      future: '{{count}} dní'
+      past: 'před {{count}} dny',
+      future: 'za {{count}} dní'
+    }
+  },
+
+  // aboutXMonths: {
+  //   one: 'about 1 month',
+  //   other: 'about {{count}} months'
+  // },
```
Comment (kossnocorp) [inline src/locale/cs/_lib/formatDistance/index.js]: Please remove commented code.

File: package-lock.json [hunk 10]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "date-fns",
+  "version": "DON'T CHANGE; IT'S SET AUTOMATICALLY DURING DEPLOYMENT; ALSO, USE YARN FOR DEVELOPMENT",
+  "lockfileVersion": 1
+}
```
Comment (kossnocorp) [inline package-lock.json]: Please remove this file to avoid the confusion, we use Yarn.

Comment (JozefBiros) [reply package-lock.json]: package-lock removed

File: src/locale/cs/_lib/formatLong/index.js (hunk 8 above)
Comment (JozefBiros) [reply src/locale/cs/_lib/formatLong/index.js]: Removed

File: src/locale/cs/_lib/localize/index.js (hunk 7 above)
Comment (JozefBiros) [reply src/locale/cs/_lib/localize/index.js]: Removed

File: src/locale/cs/_lib/localize/index.js (hunk 6 above)
Comment (JozefBiros) [reply src/locale/cs/_lib/localize/index.js]: Removed

File: src/locale/cs/_lib/localize/index.js (hunk 5 above)
Comment (JozefBiros) [reply src/locale/cs/_lib/localize/index.js]: Removed

File: src/locale/cs/_lib/match/index.js (hunk 4 above)
Comment (JozefBiros) [reply src/locale/cs/_lib/match/index.js]: Match localized

File: src/locale/cs/test.js (hunk 3 above)
Comment (JozefBiros) [reply src/locale/cs/test.js]: Removed

File: src/locale/cs/index.js (hunk 1 above)
Comment (JozefBiros) [reply src/locale/cs/index.js]: Fixed

File: src/locale/cs/index.js (hunk 2 above)
Comment (JozefBiros) [reply src/locale/cs/index.js]: Added myself.

**Comment by JozefBiros** [comment]
I addressed all isues from review, re-review please. :)
Thank you.

**Comment by kossnocorp** [comment]
I've released this PR as `v2.0.0-alpha.34`: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha34

**Comment by FredyC** [comment]
@JozefBiros Zdravim, jen se chci zeptat, nejsem tedy zadny expert na cestinu, ale neprijde mi uplne spravne pouzivat AM/PM format casu. Tedy napr. `29.4.19, 1:33 odp.`. Netvrdim, ze je to vylozene spatne, jen mam pocit, ze to neni zrovna bezne a češi spíš používají 24 format.

**Comment by JozefBiros** [comment]
@FredyC Zdravim, áno máte pravdu v CS sa používa 24h formát to som prehliadol. Predpokladám, že v CS lokalizácií bude viacero chýb. (Som SK). Pozriem sa nato keď budem mať v práci zase chvíľu času. Samozrejme PR sú výtané. :)

**Pull request merged**